slackbridge/factories.py

```diff
@@ -53,9 +53,10 @@ def add_user_bot(self, user_bot):
 
 class UserBotFactory(BotFactory):
 
-    def __init__(self, bridge_bot_factory, slack_user, channels):
+    def __init__(self, bridge_bot_factory, slack_user, users, channels):
```

Comment:
Putting this data into the shared `IRCBot` would also eliminate the need to pass this around everywhere, would be pretty nice.